.github/workflows/environment.yml

```diff
@@ -25,4 +25,8 @@ jobs:
           sudo apt clean
           docker rmi $(docker image ls -aq)
           df -h
-      - run: make environment-prepare
+      - name: prepare environment
+        run: make environment-prepare
+      - name: push image
+        if: github.event_name != 'workflow_dispatch'
```

Comment:
So we're adding a dispatch later? https://docs.github.com/en/rest/reference/actions#create-a-workflow-dispatch-event

Reply:
there's also a workflow_dispatch in there.

Reply:
Amazing.

Reply:
:)